proposals/lip-0048.md

```diff
@@ -78,43 +69,43 @@ The Fee module does not expose any functions.
 
 #### Verify Transaction
 
-During `trs` verification, the following logic is applied:
-
 ```python
-minFee = MIN_FEE_PER_BYTE * size(trs)
-         + extraCommandFee(trs.moduleID, trs.commandID)
-if trs.fee < minFee:
-     trs is invalid
+def verify(trs: Transaction) -> None:
```

Comment:
I think we agreed that it's not necessary to define types Transaction, Block, BlockHeader, they are just so common and it's clear what they mean.

Reply:
I did not notice. However I think that the Transaction type should be defined to avoid confusion, since its is not a real transaction, but we require that the params are deserialized.